pyemma/coordinates/api.py

```diff
@@ -288,7 +289,7 @@ def source(inp, features=None, top=None, chunk_size=100):
         # check: if single array, create a one-element list
         # check: do all arrays have compatible dimensions (*, N)? If not: raise ValueError.
         # create MemoryReader
-        reader = _DataInMemory(inp)
+        reader = _DataInMemory(inp, chunksize=chunk_size)
```

Comment:
Am 28.07.2015 um 16:35 schrieb Moritz Hoffmann:
> In pyemma/coordinates/api.py
> <https://github.com/markovmodel/PyEMMA/pull/458#discussion_r35654191>:
> 
>> @@ -288,7 +289,7 @@ def source(inp, features=None, top=None, chunk_size=100):
>>          # check: if single array, create a one-element list
>>          # check: do all arrays have compatible dimensions (*, N)? If not: raise ValueError.
>>          # create MemoryReader
>> -        reader = _DataInMemory(inp)
>> +        reader = _DataInMemory(inp, chunksize=chunk_size)
> 
> How do we deal with this, set chunk_size to None by default and then set
> it to 100 for anything but DataInMemory if not set otherwise?
Sounds like a practical solution, but I think chunksize=None is
confusing, because one would expect that nothing is processed chunkwise
then.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/markovmodel/PyEMMA/pull/458/files#r35654191>.
>